backend/Perflow/Services/Implementations/AlbumsService.cs

```diff
@@ -214,7 +215,7 @@ public async Task<AlbumEditDTO> UpdateEntityAsync(AlbumWriteDTO albumDTO)
 
                 album.IconURL = await _imageService.UploadImageAsync(albumDTO.Icon);
 
-                _imageService.DeleteImageAsync(oldImageId);
+                _ = _imageService.DeleteImageAsync(oldImageId);
```

Comment:
What is the purpose?

Reply:
It is to remove build warning ([CS4014](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs4014)) by explicitly discarding async operation
![image](https://user-images.githubusercontent.com/53054887/130795793-3874f10d-fc41-4378-8b7b-1863d270090b.png)